db.js

```diff
@@ -435,7 +444,25 @@ export class ActivityPubDB extends EventTarget {
     console.log(note.replies)
     if (note.replies && typeof note.replies === 'string') {
```

Comment:
```suggestion
    if (note.replies) {
```

This is why your screenshot shows you didn't load replies. iterateCollection can handle the type checking so just check if the property exists at all.